### PR TITLE
Ubus - Spectral scan in lime-openairview

### PR DESCRIPTION
**Pull request by gmarcos87**
Expose via ubus the result of the spectrum analysis obtained by fft-eval
### Usage
`ubus call lime-openairview spectral_scan '{"device":"wlan1-adhoc", "spectrum":"5ghz"`

### Output
```
{  
   "spectrum":{  
      "epoch":1550326676,
      "samples":[  
         {  
            "noise":-93,
            "central_freq":5520,
            "rssi":9,
            "data":[  
               [  
                  "5510",
                  "-108.596024"
               ],
               [  
                  "5510.356934",
                  "-106.097244"
               ],
               [  
                  "5510.714355",
                  "-108.596024"
               ],
               [  
                  "5511.071289",
                  "-102.575424"
               ],
               [  
                  "5511.428711",
                  "-100.076645"
               ],
               [  
                  "5511.785645",
                  "-96.554825"
               ],
               [  
                  "5512.143066",
                  "-106.097244"
               ],
               [  
                  "5512.5",
                  "-106.097244"
               ],
               [  
                  "5512.856934",
                  "-96.554825"
               ],
               [  
                  "5513.214355",
                  "-93.529465"
               ],
               [  
                  "5513.571289",
                  "-112.117844"
               ],
               [  
                  "5513.928711",
                  "-97.310593"
               ],
               [  
                  "5514.285645",
                  "-106.097244"
               ],
               [  
                  "5514.643066",
                  "-106.097244"
               ],
               [  
                  "5515",
                  "-118.138443"
               ],
               [  
                  "5515.356934",
                  "-108.596024"
               ],
               [  
                  "5515.714355",
                  "-108.596024"
               ],
               [  
                  "5516.071289",
                  "-112.117844"
               ],
               [  
                  "5516.428711",
                  "-101.236481"
               ],
               [  
                  "5516.785645",
                  "-102.575424"
               ],
               [  
                  "5517.143066",
                  "-104.159042"
               ],
               [  
                  "5517.5",
                  "-106.097244"
               ],
               [  
                  "5517.856934",
                  "-98.138443"
               ],
               [  
                  "5518.214355",
                  "-106.097244"
               ],
               [  
                  "5518.571289",
                  "-106.097244"
               ],
               [  
                  "5518.928711",
                  "-108.596024"
               ],
               [  
                  "5519.285645",
                  "-100.076645"
               ],
               [  
                  "5519.643066",
                  "-102.575424"
               ],
               [  
                  "5520",
                  "-103.331192"
               ],
               [  
                  "5520.356934",
                  "-104.159042"
               ],
               [  
                  "5520.714355",
                  "-106.097244"
               ],
               [  
                  "5521.071289",
                  "-102.575424"
               ],
               [  
                  "5521.428711",
                  "-108.596024"
               ],
               [  
                  "5521.785645",
                  "-112.117844"
               ],
               [  
                  "5522.143066",
                  "-101.236481"
               ],
               [  
                  "5522.5",
                  "-99.053596"
               ],
               [  
                  "5522.856934",
                  "-112.117844"
               ],
               [  
                  "5523.214355",
                  "-99.053596"
               ],
               [  
                  "5523.571289",
                  "-96.554825"
               ],
               [  
                  "5523.928711",
                  "-108.596024"
               ],
               [  
                  "5524.285645",
                  "-96.554825"
               ],
               [  
                  "5524.643066",
                  "-95.859573"
               ],
               [  
                  "5525",
                  "-102.575424"
               ],
               [  
                  "5525.356934",
                  "-108.596024"
               ],
               [  
                  "5525.714355",
                  "-100.076645"
               ],
               [  
                  "5526.071289",
                  "-98.138443"
               ],
               [  
                  "5526.428711",
                  "-112.117844"
               ],
               [  
                  "5526.785645",
                  "-106.097244"
               ],
               [  
                  "5527.143066",
                  "-98.138443"
               ],
               [  
                  "5527.5",
                  "-104.159042"
               ],
               [  
                  "5527.856934",
                  "-104.159042"
               ],
               [  
                  "5528.214355",
                  "-104.159042"
               ],
               [  
                  "5528.571289",
                  "-100.076645"
               ],
               [  
                  "5528.928711",
                  "-108.596024"
               ],
               [  
                  "5529.285645",
                  "-106.097244"
               ],
               [  
                  "5529.643066",
                  "-108.596024"
               ]
            ],
            "tsf":2147483647
         }
      ]
   }
}
```

This pull also includes some small modifications in other ubus-lime-* dependencies that I noticed wrong when trying the software.

### CODE REVIEW

File: packages/ubus-lime-batman-adv/Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@ define Package/$(PKG_NAME)
   MAINTAINER:=Marcos Gutierrez <gmarcos87@gmail.com>
   SUBMENU:=3. Applications
   TITLE:=B.A.T.M.A.N.-Adv ubus status module
-  DEPENDS:= +lua +libubox-lua +libubus-lua +kmod-batman-adv +batctl +luci-lib-nixio +lime-system
+  DEPENDS:= +lua +libubox-lua +libubus-lua +luci-lib-nixio +lime-system
```
Comment (G10h4ck) [inline packages/ubus-lime-batman-adv/Makefile]: IT is ok to remove the dependency here?

**Comment by gmarcos87** [comment]
Its ok for now

File: packages/ubus-lime-batman-adv/Makefile (hunk 1 above)
Comment (egon0) [reply packages/ubus-lime-batman-adv/Makefile]: just stumbled over this commit. it's definitive not ok to remove this dependency, afais kmod-batman-adv and batctl are not pulled anywhere else and to me it seems like a module called ubus-lime-batman-adv really needs both dependencies to work ;)

Comment (G10h4ck) [reply packages/ubus-lime-batman-adv/Makefile]: @gmarcos87 would you like to argument on this?

Comment (gmarcos87) [reply packages/ubus-lime-batman-adv/Makefile]: What I want to do is that this package will be selected automatically if lime-proto-batadv is selected. How would that be?

**Comment by G10h4ck** [comment]
> Please be aware that the fft-eval package is still only available for a single distro. Please create a PR against upstream

It would be great if you could do the effort to have it up-streamed ;)

**Pull request merged**